WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m

```diff
@@ -249,12 +253,20 @@ - (BOOL)fb_pressButton:(NSString *)buttonName error:(NSError **)error
              withDescriptionFormat:@"The button '%@' is unknown. Only the following button names are supported: %@", buttonName, supportedButtonNames]
             buildError:error];
   }
-  [[XCUIRemote sharedRemote] pressButton:remoteButton];
+
+  (duration < 0)
```

Comment:
I usually prefer to provide nil in such case and make duration of `nullable NSNumber *` type